packages/runtime/container-runtime/src/gc/garbageCollection.ts

```diff
@@ -833,7 +833,7 @@ export class GarbageCollector implements IGarbageCollector {
 			gcFeatureMatrix: this.configs.persistedGcFeatureMatrix,
 			sessionExpiryTimeoutMs: this.configs.sessionExpiryTimeoutMs,
 			sweepEnabled: false, // DEPRECATED - to be removed
-			sweepTimeoutMs: this.configs.sweepTimeoutMs,
+			tombstoneTimeoutMs: this.configs.tombstoneTimeoutMs,
```

Comment:
Don't we need to write `sweepTimeoutMs` for a while so that it doesn't disable tombstone for existing document while this version is propagated?
If a document that has `sweepTimeoutMs` opens with this version, it will write `tombstoneTimeoutMs`. The same document can be opened with an older version after this and not find `sweepTimeoutMs` and disable tombstone. It's a corner case so maybe fine to ignore it?
The other consequence is that sweepReady* and tombstoneReady* events won't be logged if a document is opened in a older version after summarizing with this version.

Reply:
Interesting point, I didn't think about that side of it.  So this is not about an existing doc with `sweepTimeoutMs`, but rather docs that have been summarized by the new code and now have only `tombstoneTimeoutMs` being opened by old runtime code.  That old code will not find `sweepTimeoutMs`... and backfill it based on Session Expiry, and that session will operate properly as well, but it will persist `sweepTimeoutMs` again... so in these cases the metadata will thrash between those two property names snapshot-to-snapshot but in each session it will have the right info and do the right thing.  Seems ok to leave as-is.

Reply:
Oh, I didn't realize that sweep timeout is backfilled for documents that don't have it in the metadata. Yeah, this sounds good then.

Reply:
Can you add a test for this if it doesn't already exist.

Reply:
Well, I can't easily test what the old code will do, all at once, but here's what we got:
- Before this PR, there's a test covering the backfill of `sweepTimeoutMs` from Session Expiry
- With this PR, there's a test covering the backfill of `tombstoneTimeoutMs` from Session Expiry
- With this PR, there's a test covering the backfill of `tombstoneTimeoutMs` from `sweepTimeoutMs`

So I think we're good.  We know any session before or after this will backfill the timeout if the expected prop is missing, and we know that the new code will upgrade to the new key.  Once saturation is reached, it'll be gravy.